### ISSUE TITLE
Double drag hands in edge case

### ISSUE THREAD

**Issue by misaugstad** [ProjectSidewalk/SidewalkWebpage#1187]
Look what I found!
![doublehands2](https://user-images.githubusercontent.com/6518824/33675572-01e8829e-da81-11e7-8bc1-e8477e0138df.png)
![doublehands](https://user-images.githubusercontent.com/6518824/33675823-dacd7006-da81-11e7-9d4a-ed43b79077bf.png)

This happens only in an edge case. Steps to reproduce:
1. Go to the audit page as an anonymous user
2. Click the 'Let's get started!' button
3. Click 'Sign in' and then sign up for a new account
4. Continue to the tutorial again

And now we get a copy of the hand. This happens at all parts of the tutorial where the hand shows up :) This does _not_ seem to happen when if you just sign in to a preexisting account instead of signing up at this step. It also doesn't seem to happen if you sign up before clicking 'Let's get started!'.

**Comment by misaugstad** [comment]
Oh yeah, everything seems to function normally. No issue showed up. The hands were the only thing that I saw duplicates of, and I was able to complete the tutorial just fine, and continue auditing as usual!

**Comment by jonfroehlich** [comment]
Don't you know, this was @manaswis's special fix for those users who did not understand what the hand animation means--so we show them TWO HANDS! It's like shouting iconically! :-P

**Comment by jonfroehlich** [comment]
I just checked this and couldn't recreate, so I recommend closing. Thoughts @misaugstad?

**Comment by misaugstad** [comment]
Yes! I simplified things to reduce the opportunity for bugs by forcing a page refresh when the user signs up. So this is no longer an issue! Just failed to reproduce myself as well. Closing now!
